src/coreclr/jit/register.h

```diff
@@ -133,6 +133,180 @@ REGDEF(K7,     7+KBASE,    KMASK(7),     "k7"   )
 
 REGDEF(STK,    8+KBASE,    0x0000,       "STK"  )
 
+// Temporary workaround to avoid changing all the code that uses REG_* enum values.
```

Comment:
I wouldn't mention "temporary" in the comments... Unless we want to find this comment and laugh about it in 10 years 😉

Reply:
comment is still needed to explain why this copy-paste is needed, I am fine with removing the Temporary, just don't want to re-run the CI for it, I can remove it before merge with b-ag